doc/misc/examples/obsolete/loader.py

```diff
@@ -23,16 +23,17 @@
 #
 
 import getpass
-from glob import glob
-from itertools import cycle, groupby
-import MySQLdb as sql
 import optparse
 import os
 import os.path
 import re
-from textwrap import dedent
 import time
 import warnings
+from glob import glob
```

Comment:
The whole `doc/misc` sub-tree should be eliminated. It's totally obsolete.

Reply:
I largely agree, but I left it here for now for our own reference while improving the newer, more up-to-date documentation.  In particular, some bits (like "scheduler-foreman-rationale", "guide-to-source", etc.) cover some historical mysteries not covered elsewhere.  The tcl script to plot chunks on a 3d-sphere might still be useful to make an illustration for the model docs, etc.

We should cut this material loose bit by bit as we continue to expand the newer docs.  I tried to sweep all the obsolete stuff scatter around into this one place at least.